README.md

```diff
@@ -50,9 +50,11 @@ For a full CI run
 
 ## Release notes
 
-### 3.2.0 (dev)
+### 3.2.0 (2017-05-09)
```

Comment:
I'd consider the changes you made in the interface a BC break, so wouldn't it be better to call this version 4.0?